Medable/ResearchStack
# PATAND353: ML: overlapping text buttons displayed in the Consent review.

Fixing overlapping text in consent screen.

how to test:
1- start the app and select QA -> hybridstudy -> multinlanguage
2- in language selector, select Spanish Latam
3- start the consent task and reach consent review

Expected: buttons should not overlap
